### PR TITLE
Add tooltips to links

### PR DESCRIPTION
**Pull request by TPGPL**
used `\((http|https)://(?!(assets|osu|status|docs|old|s)\.ppy\.sh)(?!.[^ ]* ")` to find links without tooltips

*i hope it didnt miss anything 🙄*

### CODE REVIEW

**Comment by TPGPL** [comment]
didnt fix the warnings in  `osu!tourney/Prizes` as it's getting removed in #4941

**Review by TicClick — APPROVED** [review]
approving under the premise that all web sites mentioned support HTTPS.

that being said, could you remind me what or whom we are doing this for? it brings little to no practical use, as the vast majority of captions repeat themselves, and because there's already a link preview available in every browser on hover. even when a link's title differs from its caption, it still gives astonishingly little information.

> This section will show you how to remove audio from videos using [Handbrake](https://handbrake.fr/ "Handbrake") to ...
> Initially announced in a [tweet by peppy](https://twitter.com/ppy/status/603797988742336512 "Twitter") ...
> [External guide for Mechanical Keyboard](https://www.daskeyboard.com/blog/mechanical-keyboard-guide/ "Das Keyboard")

seeing that twitter.com is Twitter, and blog.ppy.sh is peppy's blog (and these aren't just examples I chose as easy targets, it's literally everywhere), I have a very, very strong feeling that we are trying to [identify the wood](https://s3.amazonaws.com/finewoodworking.s3.tauntoncloud.com/app/uploads/2016/09/06044331/Yep%2C_its_wood-main.jpg "Amazon AWS")

File: wiki/Beatmap/Packs/en.md [hunk 1]
```diff
@@ -40,8 +40,6 @@ The beatmaps packs can be found and downloaded from: [osu! Beatmaps Packs page](
 ### Torrenting
 
 - [osu! Torrent Beatmap Packs](https://osu.ppy.sh/community/forums/topics/687910)
-  - [PandoTracker](https://pandotracker.me) by [- Pandoro](https://osu.ppy.sh/users/2574057)
-- [Beatmap-Torrents](https://osu.ppy.sh/community/forums/topics/147478) (closed)
-  - [HiraOsu! - Torrents](http://osu.hiramiya.me/torrents.htm) by [Hiramiya](https://osu.ppy.sh/users/1313441)
+  - [PandoTracker](https://pandotracker.me "PandoTracker") by [- Pandoro](https://osu.ppy.sh/users/2574057)
```
Comment (peppy) [inline wiki/Beatmap/Packs/en.md:43]: Isn't this kinda thing a step backwards? I don't think we ever want to be showing tooltips with the same content as the link itself..

**Comment by TPGPL** [comment]
the idea was to make external links distinguishable from regular wiki links, as there's no native indicator for such links like on Wikipedia

image

even if the link text is self-explanatory and it's obvious that the link directs to external sites, I don't really see any harm in adding those indicators to ensure better clarity ¯\_(ツ)_/¯

**Comment by TicClick** [comment]
if it's the only purpose, IMO a better decision would be to add the external link indicator to the part of the website responsible for wiki and get rid of all the link titles, altogether, `f o r e v e r`

**Comment by TPGPL** [comment]
it seems its the only reason, yeah, as there's no notable explanation given in the ASC

i will mark the PR as blocked for now, and remove the tooltips if thats the solution we wanna go with

**Comment by TPGPL** [comment]
superseded by #6060 i guess ¯\_(ツ)_/¯

**Pull request closed without merging**